vivarium/core/store.py

```diff
@@ -1006,6 +1006,29 @@ def emit_data(self):
             return self.value
         return None
 
+    def toggle_emits(self, emit=False, paths=None):
+        """
+        Turn on/off emits for all inner nodes of the list of paths.
+        """
+        if paths:
+            assert isinstance(paths, list), 'paths must be a list'
```

Comment:
I think the only real requirement is that `paths` be iterable, which is already going to be enforced by the `for` loop below